vllm/model_executor/models/interfaces.py

```diff
@@ -809,3 +809,56 @@ def supports_v0_only(
     model: Union[type[object], object],
 ) -> Union[TypeIs[type[SupportsV0Only]], TypeIs[SupportsV0Only]]:
     return getattr(model, "supports_v0_only", False)
+
+
+@runtime_checkable
+class SupportsEagle3(Protocol):
+    """The interface required for models that support 
+    EAGLE3 speculative decoding."""
+
+    supports_eagle3: ClassVar[Literal[True]] = True
+    """
+    A flag that indicates this model supports EAGLE3 
+    speculative decoding.
+
+    Note:
+        There is no need to redefine this flag if this class is in the
+        MRO of your model class.
+    """
+
+    def set_aux_hidden_state_layers(self, layers: tuple[int]) -> None:
```

Comment:
```suggestion
    def set_aux_hidden_state_layers(self, layers: tuple[int, ...]) -> None:
```

`tuple[int]` means a tuple containing a single integer, you're probably looking for `tuple[int, ...]` which means a tuple with any number of integeres

Reply:
Good catch!